### ISSUE TITLE
Error: An error occurred while trying to export the selected file.

### ISSUE THREAD

**Issue by SafetyLit** [retorquere/zotero-safetylit-mods#2]
Your newest SafetyLit MODS.js file (419 KB) will not allow an export. Thanks

**Comment by retorquere** [comment]
It does for me. Does this happen with all your items, or just some? In `Help` -> `Debug Output Logging` -> `View Output` there wille be an error message about the export, can you paste that here? I will always test the file before upload, so it will not be the case that it categorically will not export anything.

I do not see a version that is 419KB. All versions are 205K. The best way to identify a specific version is by the `lastUpdated` timestamp at the top of the file.

**Comment by SafetyLit** [comment]
I tried again today after downloading the file again and was able to export the SafetyLit MODS file. I suspect that the problem may be the way that I move the file into the  Zotero translators directory. There must be a better / more simple way than I've been doing it.  Perhaps, I reverted to your first version. I open the file from the github webpage, copy the text; paste it into my text editor (BBEdit); save the file as SafetyLit MODS.js to my downloads directory (after changing the extension from .js.txt to remove the txt part); then drag the file to the translator directory.

I fear that I am missing a simple way to do this. 

[Rhetorical question} Why does Apple add ".txt" to the extension? I don't recall having that problem before now.

I apologize for my inexperience / naiveté.
